test/near_provider.test.js

```diff
@@ -1,70 +1,44 @@
+/**
+ * These tests default to running a local NEAR node. nearcore must be
+ * running at the same folder level.
+ */
+
 const fs = require('fs');
-const os = require('os');
-const path = require('path');
 const web3 = require('web3');
-const nearlib = require('nearlib');
+const nearlib = require('near-api-js');
 const utils = require('../src/utils');
 const { NearProvider } = require('../src/index');
 
-/**------------------------------------------------
- * TESTS IN FLUX
- *
- * Need to setup making transactions
- *
- * Hashes are hardcoded. New transactions need to be made and the
- * hashesupdated whenever testnet resets.
- *
- * ------------------------------------------------
- */
-
-const NEAR_NODE_URL = 'http://localhost:3030';
-const networkId = 'local'; // see NearProvider constructor, src/index.js
 const nearEvmFile = './artifacts/near_evm.wasm';
 const zombieCodeFile = './artifacts/zombieAttack.bin';
 const zombieABIFile = './artifacts/zombieAttack.abi';
-const testNearProvider = new nearlib.providers.JsonRpcProvider(NEAR_NODE_URL);
 
-const configPath = path.resolve(os.homedir(), '.near', 'validator_key.json');
-const config = require(configPath);
+// see NearProvider constructor, src/index.js
+const NETWORK_ID = process.env.NEAR_NETWORK_ID || 'local';
```

Comment:
we actually use `NEAR_ENV` environment variable in near-shell and other places.

Reply:
Oh good to know. I'll make this change.